b2sdk/file_version.py

```diff
@@ -282,6 +311,30 @@ def __init__(
             cache_control=cache_control,
         )
 
+    @property
+    def expires(self) -> str | None:
+        return self.file_info.get('b2-expires')
+
+    def expires_parsed(self) -> dt.datetime | None:
```

Comment:
since `expires()` is `@property`, wouldnt it look more consistent to have this as `@property` too?

Reply:
I made it a property at first but then changes. It would look more consistent but it doesn't behave the same way as `expires` - in particular, it can blow up, raise an exception. PEP-008 only discourages using properties that are computationally expensive but I've seen people extending that for methods that can raise an exception, I think for a good reason. Though I could go either way here, to be honest.